lib/bundled_rbi/dry-monads.rbi

```diff
@@ -180,7 +180,7 @@ class Dry::Monads::Result
 
   sig do
     type_parameters(:New)
-      .params(blk: T.proc.params(arg0: SuccessType).returns(T.type_parameter(:out, :New)))
+      .params(blk: T.proc.params(arg0: SuccessType).returns(T.type_parameter(:New)))
       .returns(Dry::Monads::Result[FailureType, T.type_parameter(:out, :New)])
```

Comment:
Should we blast this one as well? I don't properly understand when `:out` is required 😐 but removing it from both places caused a bunch of nice type errors in V3.

Reply:
Yeah I reckon so 👍